src/python/grpcio/grpc/_cython/_cygrpc/fork_posix.pyx.pxi

```diff
@@ -82,12 +84,14 @@ cdef void __postfork_child() nogil:
 def fork_handlers_and_grpc_init():
     grpc_init()
     if _GRPC_ENABLE_FORK_SUPPORT:
-        # TODO(ericgribkoff) epoll1 is default for grpcio distribution. Decide whether to expose
-        # grpc_get_poll_strategy_name() from ev_posix.cc to get actual polling choice.
-        if _GRPC_POLL_STRATEGY is not None and _GRPC_POLL_STRATEGY != "epoll1":
+        # TODO(ericgribkoff) epoll1 is default for grpcio distribution (poll is
```

Comment:
When compiled from source (from git or from pip installing source package) we get `epollex` by default on Linux. `manylinux1` binaries are on `epoll1`.

Reply:
Good point: I moved the check for this into core's pre-fork handler, where it knows the actual poll strategy in use.